src/taipy/rest/api/resources/pipeline.py

```diff
@@ -197,37 +197,13 @@ class PipelineList(Resource):
     def __init__(self, **kwargs):
         self.logger = kwargs.get("logger")
 
-    def fetch_config(self, config_id):
-        config = Config.pipelines.get(config_id)
-        if not config:
-            raise NonExistingPipelineConfig(config_id)
-        return config
-
     @_middleware
     def get(self):
         schema = PipelineResponseSchema(many=True)
         manager = _PipelineManagerFactory._build_manager()
         pipelines = [_to_model(REPOSITORY, pipeline) for pipeline in manager._get_all()]
         return schema.dump(pipelines)
```

Comment:
Since we are going to remove the Pipeline entity soon, this method will be removed as well right?

Reply:
I will say yes, but I think the topic deserves some more discussions :D I think Pipeline can be a hidden class so this API will be removed as we won't expose it anymore. But it's still up for discussion